media-api/app/lib/ImageExtras.scala

```diff
@@ -40,29 +40,46 @@ object ImageExtras {
   import scala.concurrent.duration._
   import com.gu.mediaservice.lib.FeatureToggle
 
-  def validityMap(image: Image): Map[String, Boolean] = Map(
-    "paid_image"           -> Costing.isPay(image.usageRights),
-    "conditional_paid"     -> Costing.isConditional(image.usageRights),
-    "no_rights"            -> !hasRights(image.usageRights),
-    "missing_credit"       -> !hasCredit(image.metadata),
-    "missing_description"  -> !hasDescription(image.metadata),
-    "current_deny_lease"   -> hasCurrentDenyLease(image.leases),
-    "over_quota"           -> UsageQuota.isOverQuota(image.usageRights)
+  case class ValidityCheck(valid: Boolean, overrideable: Boolean)
+
+  def validityMap(image: Image): Map[String, ValidityCheck] = Map(
+    "paid_image"           -> ValidityCheck(Costing.isPay(image.usageRights), true),
+    "conditional_paid"     -> ValidityCheck(Costing.isConditional(image.usageRights), true),
+    "no_rights"            -> ValidityCheck(!hasRights(image.usageRights), true),
+    "missing_credit"       -> ValidityCheck(!hasCredit(image.metadata), false),
+    "missing_description"  -> ValidityCheck(!hasDescription(image.metadata), false),
+    "current_deny_lease"   -> ValidityCheck(hasCurrentDenyLease(image.leases), true),
+    "over_quota"           -> ValidityCheck(UsageQuota.isOverQuota(image.usageRights), true)
   )
```

Comment:
The logic is:
- Images can be invalid -> Invalid images can't be cropped
- Images can be deemed invalid by a set of checks
- Image validity can be overridden (to valid) if any one of a set of checks is met
- Some of the set of checks that mark an image as invalid cannot be overridden